server/server.go

```diff
@@ -195,6 +210,12 @@ func NewMCPServer(
 		version:              version,
 		notificationHandlers: make(map[string]NotificationHandlerFunc),
 		notifications:        make(chan ServerNotification, 100),
+		capabilities: serverCapabilities{
```

Comment:
I think the default for this should be empty. We shouldn't force anyone to implement these capabilities by default.

Reply:
@ezynda3 You are right; this way is more reasonable. I have already modified my latest commit in this way, but I have another concern: the default value here was copied from the current version. After adjusting the default value now, it equals changing the default behavior of the library. Perhaps a breaking change notification is needed, and you might need to consider this issue.